<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QThreadPoolClass</class>
 <widget class="QWidget" name="QThreadPoolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QThreadPool</string>
  </property>
  <widget class="QLabel" name="num_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>25</y>
     <width>471</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>161</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>生成随机数</string>
   </property>
   <widget class="QListWidget" name="rand_listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QThreadPool.qrc"/>
 </resources>
 <connections/>
</ui>
